src/main/scala/dicom/DicomDeidentifyDictionary.scala

```diff
@@ -98,15 +93,16 @@ object DicomDeidentifyDictionary {
             name = rowCellTexts(0),
             keyword = Keyword.valueOf(intTag),
             action = getActionCode(rowCellTexts(4)).getOrElse(ActionCode.X),
-            retainUidsAction = getActionCode(rowCellTexts(6)),
-            retainDevIdAction = getActionCode(rowCellTexts(7)),
-            retainInstIdAction = getActionCode(rowCellTexts(8)),
-            retainPatCharsAction = getActionCode(rowCellTexts(9)),
-            retainLongFullDatesAction = getActionCode(rowCellTexts(10)),
-            retainLongModifDatesAction = getActionCode(rowCellTexts(11)),
-            cleanDescAction = getActionCode(rowCellTexts(12)),
-            cleanStructContAction = getActionCode(rowCellTexts(13)),
-            cleanGraphAction = getActionCode(rowCellTexts(14))
+            deidOptionToAction = DeidOption.values
+              .zip(
+                Range(14, 5, -1).map(colIdx =>
+                  getActionCode(rowCellTexts(colIdx))
+                )
+              )
+              .collect({ case (option, Some(actionCode)) =>
+                (option, actionCode)
```

Comment:
When building Map, I usually prefer writing 

```suggestion
                option -> actionCode
```

which does exactly the same (create a Tuple2), but IMO following this convention makes for a bit more readable code